pytest.ini

```diff
@@ -2,4 +2,5 @@
 addopts = -svx
 testpaths =
     tests/
-    slow_tests/
+markers =
+    memcheck: marks tests to be run with cuda memory checks, triggers cuda device reset at end of marked tests (deselect with '-m "not memcheck"')
```

Comment:
not directly related to the PR, and not sure if really desirable, but just noting that it looks like pytest config can be [put in pyproject.toml](https://docs.pytest.org/en/6.2.x/customize.html#pyproject-toml)